base_layer/wallet/tests/transaction_service/service.rs

```diff
@@ -930,7 +930,7 @@ fn test_accepting_unknown_tx_id_and_malformed_reply<T: TransactionBackend + Clon
             futures::select! {
                 event = alice_event_stream.select_next_some() => {
                     if let TransactionEvent::Error(s) = &*event.unwrap() {
-                        if s == &"TransactionError(ValidationError(\"Transaction could not be finalized\"))".to_string() {
+                        if s == &"TransactionProtocolError(TransactionBuildError(InvalidSignatureError))".to_string() {
```

Comment:
This is a sign that whoever wrote this initially (yes it was me :P) didn't do this right for maintainability reasons.

Reply:
All good actually :-)